lib/analyzer/os-release/release-analyzer.ts

```diff
@@ -76,7 +76,10 @@ export async function tryRedHatRelease(
   if (!idRes || !versionRes) {
     throw new Error("Failed to parse /etc/redhat-release");
   }
-  const name = idRes[1].replace(/"/g, "").toLowerCase();
+  const name =
+    text.indexOf("Red Hat") !== -1
```

Comment:
Looking at the present parsing logic I cannot see how this could ever have worked for RedHat before version 7 (I think the parsing logic might have only been tested on CentOS). Since we need the value `rhel` for the os name when we have an actual RedHat image I couldn't think a better fix.

Reply:
I think this is right, I also can't think of a better way. Maybe they've also changed the older images like centos? Who knows...